webapp/src/components/rhs/rhs_checklist_list.tsx

```diff
@@ -212,6 +214,15 @@ const RHSChecklistList = (props: Props) => {
             <MainTitleBG numChecklists={checklists.length}>
                 <MainTitle>
                     {formatMessage({defaultMessage: 'Checklists'})}
+                    {
+                        overdueTasksNum > 0 &&
+                        <OverdueTasksToggle
+                            toggled={checklistItemsFilter.overdueOnly}
+                            onClick={() => selectOption('overdueOnly', !checklistItemsFilter.overdueOnly)}
+                        >
+                            {formatMessage({defaultMessage: '{num} tasks overdue'}, {num: overdueTasksNum})}
```

Comment:
You can use the [plural format](https://formatjs.io/docs/core-concepts/icu-syntax#plural-format) here to cover the case where there is a single task overdue:
```suggestion
                            {formatMessage({defaultMessage: '{num} {num, plural, =1 {task} other {tasks}} overdue'}, {num: overdueTasksNum})}
```

Reply:
Great catch! Also didn't know about plural support, thanks ;)